scenarios/kubernetes_e2e_test.py

```diff
@@ -105,6 +108,16 @@ def test_docker(self):
         for call in self.callstack:
             self.assertTrue(call.startswith('docker'))
 
+    def test_default_tag(self):
+        """Ensure the default tag exists on gcr.io."""
+        args = self.parser.parse_args()
+        image = kubernetes_e2e.kubekins(args.tag)
+        m = re.match('gcr.io/([^:]+):(.+)', kubernetes_e2e.kubekins(args.tag))
+        self.assertIsNotNone(m)
+        url = 'https://gcr.io/v2/%s/manifests/%s' % (m.group(1), m.group(2))
+        data = json.loads(urllib.urlopen(url).read())
```

Comment:
Would `docker pull kubekins:tag` be simpler and more complete here?

There could be an issue with the intermediate images, for example.